src/language/code_generator.py

```diff
@@ -25,14 +24,20 @@
 
 LOGGING_FORMAT = "%(levelname)s:%(name)s: %(message)s"
 LOGGER = logging.getLogger("NMODLCodeGen")
+CLANG_FORMAT_EXECUTABLE = (
```

Comment:
a pure nitpick: it's not  really a`clang-format` executable?

Reply:
`CLANG_FORMAT_COMMAND` then :)